### DISCUSSION TITLE
Is "BeeSync" a rip-off of AFFiNE?

### DISCUSSION THREAD

**Discussion by JokerQyou** [toeverything/AFFiNE#9010] [General]
Just found this thing on social media: https://github.com/BeeSyncAI/BeeSync?tab=readme-ov-file#contributors . It has some trace of "affine" in codebase (search shows link to `affine.pro` in README), does not mention it's a fork, and hosts a SaaS offer. Does that violate AFFiNE license?

**Comment by Brooooooklyn** [comment]
> Does that violate AFFiNE license?

Yes, thank you for letting us know. We will look at how to report this repository.
